### ISSUE TITLE
Supported Raspberries

### ISSUE THREAD

**Issue by capsload2** [Gabboxl/ts3serverARM#1]
Will
This also work on 3 A+ ?

Cheers

**Comment by capsload2** [comment]
Or even raspb 2

**Comment by Gabboxl** [comment]
Hi, 
Yes, it should work - maybe a little slower than the 3B+ because of the cpu but it should.

If you can try the guide on these two models I'll add a section with the models tested :)

**Comment by capsload2** [comment]
Hey!

I've installed everything using TeamSpeak 3.12.1 my Raspberry 3A+

When i start, everything works until:

2020-09-18 10:10:33.667376|INFO    |Accounting    |   |max slots         : 32

After a while (20-30 seconds), i get a:

Bus error

Could this be, because i use Raspberry OS, or a outdated qemu version?

**Comment by Gabboxl** [comment]
> Could this be, because i use Raspberry OS, or a outdated qemu version?

If you are using the 32-bit version of Raspberry OS, maybe that's the problem. Try following the guide on a 64-bit OS, like Ubuntu Server for rpi (this is the one that I use).

Oh and yes, make sure you are using qemu 4.0.0 or newer.

**Comment by capsload2** [comment]
Trying it with the Raspberry OS 64-bit Beta!

**Comment by capsload2** [comment]
It finally worked for me with my Raspberry 3 A+ and Ubuntu 20.04.1, 64 Bit

It didn't worked with Raspbarian OS 64 Bit. (I couldnt get qemu etc. installed).

Pro Tip: A automatic installer.sh that does all the work would be awesome!

**Comment by Gabboxl** [comment]
@capsload2 Nice! Maybe qemu isn't in Raspberry Pi OS' repos.
I was thinking to make an automatic script too, I'll see next week.
Thank you for the testing job!
Can I consider this issue closed?

**Comment by capsload2** [comment]
Yep thanks!

**Comment by capsload2** [comment]
Any way so Raspberry TS3 Server is automatically starting after a boot of the Raspberry?

**Comment by Gabboxl** [comment]
I read your comment about the "Unsupported ancillary data: 0/8" text spammed in the console, I'm aware of this issue but I didn't find any solution for this. 
Anyway, you have to create a service with a script to start automatically the server on boot.

**Comment by capsload2** [comment]
Do you have any great script that already let me Raspberry start TS3 after a boot ;P ?

**Comment by Gabboxl** [comment]
I'm not at home at the moment, I'll expand this guide next week with your suggestions! 
Can I credit you for testing the guide on rpi 3A+?

**Comment by capsload2** [comment]
Sure! That'd be great :)

**Comment by capsload2** [comment]
Did you had time yet for the auto-run script after booting the Rasp? :)

**Comment by Gabboxl** [comment]
Hello!  I have just updated the guide, sorry for the wait... busy days!

**Comment by capsload2** [comment]
Excellent!

Now to make it more selfie: a ts3 automatic server-updater integration would be awesome as well.

What would be nice is now for the final part, a "one-liner" command that does everything by its own step by step.

curl script.sh | bash script.sh 

and everything should be installed, configured and autostarted ;-)

**Comment by capsload2** [comment]
im a complete newb ;P but i just started with smth.

```
echo "we recommend using this as root user"
apt-get install qemu qemu-user qemu-user-static debootstrap binfmt-support binutils -y
echo "complete, moving on"
debootstrap --arch i386 stretch ./chroot-stretch http://ftp.us.debian.org/debian
echo "complete, moving on"
cp /usr/bin/qemu-i386-static ./chroot-stretch/usr/bin/
echo "complete, starting environment"
mount -t sysfs sys ./chroot-stretch/sys/
echo "step 1 done"
mount -t proc proc ./chroot-stretch/proc/
echo "step 2 done"
mount --bind /dev ./chroot-stretch/dev/
echo "step 3 done"
mount --bind /dev/pts ./chroot-stretch/dev/pts/
echo "step 4 done"
mount --bind /dev/shm ./chroot-stretch/dev/shm/
echo "all steps done. continue with -- Entering in the x86 environment --"
```

**Comment by Gabboxl** [comment]
Yeah, this is a great way to start! I'll update the guide with the script for creating the virtual environment once done. I don't plan to write a script for creating the ts3 server nor for updating it (maybe there's already something on the internet! see https://github.com/Gabboxl/ts3serverARM/issues/3).
